release_notes.md

```diff
@@ -3,21 +3,9 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR)
 -->
+- Added fallback behavior to ensure in-proc payload compatibility with "dotnet-isolated" as the `FUNCTIONS_WORKER_RUNTIME` value (#10439)
 - Update Java Worker Version to [2.17.0](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.17.0)
   - Update application insights agent version to 3.5.4
   - Includes fixes from 2.16.0
-- Update Python Worker Version to [4.31.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.31.0)
```

Comment:
Why are we removing these entries?

Reply:
Because it was part of 4.1036.1 https://github.com/Azure/azure-functions-host/releases/tag/v4.1036.1

Reply:
We are adding only changes from 4.1036.1 to .2